modules/tsconfig.shared.json

```diff
@@ -16,7 +16,9 @@
     "types": ["reflect-metadata", "bluebird-global"],
     "paths": {
       "common/*": ["../../src/bp/common/*"],
-      "botpress/sdk": ["../../src/bp/sdk/botpress.d.ts"]
+      "botpress/sdk": ["../../src/bp/sdk/botpress.d.ts"],
+      "nlu-core": ["../../src/bp/nlu-core"],
```

Comment:
**pointing the attention of reviwers on this line**

Reply:
That's not necessary. Please put your typings in `common/nlu-core.typings.ts`. There's no reason to add a new package to the tsconfig, everything in `common` is available for the core, modules (backend/frontend), and studio/admin.

Reply:
mmmhhhh, I disagree with you on this one. I don't think NLU code belongs inside `common/`.

We're now a standalone team and we need to move freely with a "single responsibility" package.

Reply:
We will eventually split the codebase, but for now let's follow what we already have and keep it simple. We didn't discuss about the new structure yet